src/sentry/templatetags/sentry_react.py

```diff
@@ -95,10 +95,15 @@ def get_react_config(context):
         messages = []
         is_superuser = False
 
+    enabled_features = []
+
     if user:
         user = extract_lazy_object(user)
+    if not user.is_anonymous():
+        enabled_features = [project.organization.slug + ':' + project.slug + ':similarity-view'
+            for project in Project.objects.filter(organization=User.objects.get(id=user.id).get_orgs()) if
+            features.has('projects:similarity-view', project=project)]
```

Comment:
And to be clear, there are relatively expensive ways we can do this. You can do like:

```
projects = set()
for organization in Organization.objects.get_for_user(user):
    for team in Team.objects.get_for_user(organization, user):
        projects.update(Project.objects.get_for_user(team, user, _skip_team_check=True))
```

But to me, this just feels like you're going against the grain since this isn't a pattern that we do anywhere else and we have nothing optimized for it.